ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-rollback-nightly.yaml

```diff
@@ -0,0 +1,191 @@
+base_images:
```

Comment:
We already have too many config files (150+), how about merge them into *__amd64-nightly.yaml  ?

Reply:
They are rollback ci, they are a bit different with the regular upgrade ci, have to introduce some more release image, I prefer to not bother the existing regular upgrade ci yaml. And we already have existing stable rollback ci yaml, this time it is adding nightly rollback ci yaml to have a corresponding pair.